marathon/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## 1.3.1 / Unreleased
 
 * [BUGFIX] Fix missing task count metrics (tasksStaged, tasksRunning, tasksHealthy, tasksUnhealthy) when filtering by group.
+* [BUGFIX] Fixes `marathon.queue.count` metric to report 0 for apps that have no undeployed instances.
```

Comment:
We've an automatic job for this now, no need to manually update the changelog, PR title will be used at release time.